lib/matplotlib/cbook/__init__.py

```diff
@@ -2242,21 +2242,39 @@ def type_name(tp):
                     type_name(type(v))))
 
 
-def _check_in_list(_values, **kwargs):
+def _check_in_list(_values, *, _print_supported_values=True, **kwargs):
     """
     For each *key, value* pair in *kwargs*, check that *value* is in *_values*;
     if not, raise an appropriate ValueError.
```

Comment:
Can drop this line since it's in `Raises` below .
```suggestion
    For each *key, value* pair in *kwargs*, check that *value* is in *_values*.
```